experiments/standalone/neural_snow/display_tools.jl

```diff
@@ -0,0 +1,110 @@
+using Plots
```

Comment:
For now, let's move this to where it is used, and leave the Project.toml and Manifest.toml for the experiments section unchanged.